daemon/common_test.go

```diff
@@ -1,22 +1,25 @@
 package daemon
 
 import (
+	"bytes"
 	"encoding/hex"
 	"fmt"
 	"os"
 	"path/filepath"
 
-	"github.com/bblfsh/bblfshd/daemon/protocol"
-	"github.com/bblfsh/bblfshd/runtime"
-
 	"github.com/containers/image/types"
-	oldctx "golang.org/x/net/context"
+	xcontext "golang.org/x/net/context"
```

Comment:
Do we still support any Go releases old enough that this couldn't be more simply `"context"` in the standard section above?